.github/workflows/pitest_two.yml

```diff
@@ -0,0 +1,106 @@
+name: "Pitest2"
```

Comment:
Please make name more meaningful, smth like "Pitest - main, javadoc and indentation packages"
Same for other files

Reply:
No, more required.